script/import_scripts/base/generic_database.rb

```diff
@@ -329,6 +439,34 @@ def create_topic_table
       @db.execute 'CREATE UNIQUE INDEX IF NOT EXISTS topic_upload_unique ON topic_upload(topic_id, path)'
     end
 
+    def create_pm_topic_table
+      @db.execute <<-SQL
+        CREATE TABLE IF NOT EXISTS pm_topic (
+          id #{key_data_type} NOT NULL PRIMARY KEY,
+          title TEXT,
+          raw TEXT,
+          category_id #{key_data_type},
+          closed BOOLEAN NOT NULL DEFAULT false,
+          user_id #{key_data_type} NOT NULL,
+          target_users TEXT,
+          created_at DATETIME,
+          url TEXT,
+          upload_count INTEGER DEFAULT 0
+        )
+      SQL
+
+      @db.execute 'CREATE INDEX IF NOT EXISTS topic_by_user_id ON topic (user_id)'
+
+      @db.execute <<-SQL
+        CREATE TABLE IF NOT EXISTS topic_upload (
+          topic_id #{key_data_type} NOT NULL,
+          path TEXT NOT NULL
+        )
+      SQL
+
+      @db.execute 'CREATE UNIQUE INDEX IF NOT EXISTS topic_upload_unique ON topic_upload(topic_id, path)'
```

Comment:
I think the index and table aren't needed, otherwise it should reference the `pm_topic` table.